cmd/snap/cmd_info.go

```diff
@@ -76,6 +76,40 @@ func init() {
 		}), nil)
 }
 
+func (iw *infoWriter) maybePrintHealth() {
+	if iw.localSnap == nil {
+		return
+	}
+	health := iw.localSnap.Health
+	if health == nil {
+		if !iw.verbose {
+			return
+		}
+		health = &client.SnapHealth{
```

Comment:
is this bit tested?

Reply:
no, there isn't coverage for this particular bit

Reply:
now there is